JavaScript/JavaScript1/Week3/noteApp.js

```diff
@@ -0,0 +1,56 @@
+// NOnoN0nOYes (Note taking app)
+const notes = [];
+
+// Save the note
+function saveNote(content, id) {
+  const note = {
+    content: content,
+    id: id,
```

Comment:
When key and value are the same, you can shorten it like this:
```suggestion
    content,
    id,
```